code/obj/item/book.dm

```diff
@@ -269,15 +269,11 @@ Custom Books
 		else
 			if(ishuman(hit_atom))
 				var/mob/living/carbon/human/user = usr
+				var/mob/living/carbon/human/H = hit_atom
 				var/hos = (istype(user.head, /obj/item/clothing/head/hosberet) || istype(user.head, /obj/item/clothing/head/hos_hat))
-				if(hos)
-					var/mob/living/carbon/human/H = hit_atom
-					H.changeStatus("stunned", 2 SECONDS)
-					H.changeStatus("weakened", 2 SECONDS)
-					H.force_laydown_standup()
-					//H.paralysis++
-					playsound(H.loc, "swing_hit", 50, 1)
-					usr.say("I AM THE LAW!")
+				if(hos && !GET_COOLDOWN(H, "spacelaw_confession")
+					H.say("[pick("Alright, fine, I ", "I confess that I ", "I confess! I ", "Okay, okay, I admit that I ")][pick("nabbed ", "stole ", "klepped ", "grabbed ", "thieved ", "pilfered ")]the [pick("Head of Security's ", "Captain's ", "Head of Personnel's ", "Chief Engineer's ", "Research Director's ", "Heisenbee's ", "Science Department's ", "Mining Team's ", "Quartermaster's ")] [pick("hair brush!", "shoes!", "stuffed animal!", "spare uniform!", "bedsheets!", "hat!", "trophy!", "glasses!", "fizzy lifting drink!", "ID card!")]")
+					ON_COOLDOWN(H, "spacelaw_confession", 10 SECONDS)
```

Comment:
```suggestion
				if(hos && !ON_COOLDOWN(H, "spacelaw_confession", 10 SECONDS)
					H.say("[pick("Alright, fine, I ", "I confess that I ", "I confess! I ", "Okay, okay, I admit that I ")][pick("nabbed ", "stole ", "klepped ", "grabbed ", "thieved ", "pilfered ")]the [pick("Head of Security's ", "Captain's ", "Head of Personnel's ", "Chief Engineer's ", "Research Director's ", "Heisenbee's ", "Science Department's ", "Mining Team's ", "Quartermaster's ")] [pick("hair brush!", "shoes!", "stuffed animal!", "spare uniform!", "bedsheets!", "hat!", "trophy!", "glasses!", "fizzy lifting drink!", "ID card!")]")
```

Also, Heisenbee will have a grammatical error in there (" ... the Heisenbee's ...")

Reply:
fixed and removed the heisenbee one, it'd be too much of a hassle to work around